Assets/MixedRealityToolkit.Extensions/QRCodesTracker/Scripts/QRCodesManager.cs

```diff
@@ -0,0 +1,297 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// The process for distributing QRCodesTrackerPlugin.dll is not yet defined. This code will remain unusable
+// to the general public until said distribution story is determiend. However, this file has been added to enable
+// public facing development.
+// #define QRCODESTRACKER_BINARY_AVAILABLE
+
+#if QRCODESTRACKER_BINARY_AVAILABLE
+
+#if UNITY_WSA
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+#if WINDOWS_UWP
+using Windows.Perception.Spatial;
+using Windows.Perception.Spatial.Preview;
+#endif
+
+using QRCodesTrackerPlugin;
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.QRCodesTracker
+{
+    public static class QRCodeEventArgs
+    {
+        public static QRCodeEventArgs<TData> Create<TData>(TData data)
+        {
+            return new QRCodeEventArgs<TData>(data);
+        }
+    }
+
+    [Serializable]
+    public class QRCodeEventArgs<TData> : EventArgs
+    {
+        public TData Data { get; private set; }
+
+        public QRCodeEventArgs(TData data)
+        {
+            Data = data;
+        }
+    }
+
+    public class QRCodesManager : MonoBehaviour
+    {
+        [Tooltip("Determines if the QR codes scanner should be automatically started.")]
+        public bool AutoStartQRTracking = true;
+
+        public bool IsTrackerRunning { get; private set; }
+        public QRCodesTrackerPlugin.QRTrackerStartResult StartResult { get; private set; }
+
+        public event EventHandler<bool> QRCodesTrackingStateChanged;
```

Comment:
>EventHandler<bool> [](start = 21, length = 18)

Or why not simply employ this approach and do EventHandler<QRCodesTrackerPlugin.QRCode>?

Reply:
I will address qrcodesmanager changes in a separate review. This code is currently being worked on in a different repo.